### PR TITLE
Consistent policy format via conftest fmt

### PR DESCRIPTION
**Pull request by mrueg**
@jalseth  
As requested, I added a github check that will fail if it is not consistent.

Fixes: #238

### CODE REVIEW

File: examples/container-deny-added-caps/src_test.rego [hunk 1]
```diff
@@ -1,22 +1,13 @@
 package container_deny_added_caps
 
 test_dropped_all {
-    input := {
-        "securityContext": {
-            "capabilities": {"drop": ["alL"]}
```
Comment (mrueg) [inline examples/container-deny-added-caps/src_test.rego:6]: @jalseth is the capitalization of "alL" made on purpose or should it rather be "all"?

Comment (jpreese) [reply examples/container-deny-added-caps/src_test.rego:6]: This should be "all"

Comment (mrueg) [reply examples/container-deny-added-caps/src_test.rego:6]: Thanks, I changed it and regenerated the policies so the acceptance tests should work.

Comment (jalseth) [reply examples/container-deny-added-caps/src_test.rego:6]: It was actually intentional to verify the policy to ignore the casing of the key word, but it really doesn't matter at this point :)

**Review by jalseth — APPROVED** [review]
LGTM! I disagree with a few of the formatter's choices, but the consistency is definitely an improvement.

**Comment by jalseth** [comment]
Also thank you for putting this PR together! It's much appreciated.

**Pull request merged**